asyncapi/hubspot/Module.md

```diff
@@ -0,0 +1,59 @@
+## Overview
+
+The [Ballerina](https://ballerina.io/) listener for Hubspot provides capability to listen following events happening in HubSpot account.
+* `onCompanyCreation`
+* `onCompanyDeletion`
+* `onContactCreation`
+* `onContactDeletion`
+* `onConversationCreation`
+* `onConversationDeletion`
+* `onDealCreation`
+* `onDealDeletion`
+
+## Prerequisites
+
+Before using this connector in your Ballerina application, complete the following:
+
+- Create [Hubspot account](https://app.hubspot.com/signup-hubspot/)
+- Create an [app](https://developers.hubspot.com/get-started) in devoloper portal
+- Go to webhooks and register a targetURL for notifications.
+- Create a subscription with interested events.
+
+## Quickstart
+To use the Hubspot listener in your Ballerina application, update the .bal file as follows:
+
+### Step 1: Import listener
+Import the ballerinax/trigger.hubspot module into the Ballerina project.
+```ballerina
+    import ballerinax/trigger.hubspot;
+```
+
+### Step 2: Create a new listener instance
+Create a `listenerConfig` with the AppSecret and CallbackURL. 
```

Comment:
```suggestion
To create a `listenerConfig` with the application secret and the callback URL, add the following configuration.
```
If "configuration" is the wrong word to use here, replace it with a more appropriate word.